### DISCUSSION TITLE
New filter possibility

### DISCUSSION THREAD

**Discussion by Lony35** [amidaware/tacticalrmm#1534] [General]
Hello
I would like to be able to filter the servers/ workstations in relation to the installed software.

In my company, we often do audits to verify that the machines match a model with certain software by profile.

Only, having to look in each machine if a software is well installed, requires a lot of time.

Thank you in advance

**Comment by silversword411** [comment]
Until reporting comes out you can use the management commands to get computers with specific software if that helps. See the docs

**Comment by dinger1986** [comment]
Sorry I meant to check this earlier, you could use a powershell script to write to a custom filed which you can the filter by using the search to see if its installed or not, obviously reports will be the best solution but thats one for now which will work
